src/pdf/pdfgraphics.cpp

```diff
@@ -1860,15 +1860,15 @@ wxPdfDocument::SetLineStyle(const wxPdfLineStyle& linestyle)
   }
 
   const wxPdfArrayDouble& dash = linestyle.GetDash();
-  if (&dash != NULL)
+  if (dash.GetCount() > 0)
   {
-    wxString dashString = wxT("");
+    wxString dashString = wxS("");
     size_t j;
     for (j = 0; j < dash.GetCount(); j++)
     {
       if (j > 0)
       {
-        dashString += wxString(wxT(" "));
+        dashString += wxString(wxS(" "));

```

Comment:
Not clear to me what the differences in wxS and wxT according to the wxWidgets documentation, but this is fine.

Reply:
I copied the function exactly from wxpdfdoc upstream and the upstream maintainer made that change. I decided to keep it to minimise divergence from the upstream code.